src/main/resources/templates/fragments/results.html

```diff
@@ -25,23 +32,25 @@ <h5 class="modal-title" id="exampleModalCenterTitle">Export to CSV</h5>
                                     <span aria-hidden="true">&times;</span>
                                 </button>
                             </div>
-                            <div class="modal-body">
+
+                            <div class="modal-body" id="exportCsv" th:fragment="exportCsv">
+                                <p th:text = "${exported}"></p>
                                 <div class="card bg-light">
                                     <div class="card-header">
                                         Columns
                                     </div>
                                     <div class="card-body">
                                         <form>
                                             <div class="custom-control custom-checkbox">
-                                                <input type="checkbox" class="custom-control-input" id="courseCheck" checked>
+                                                <input type="checkbox" class="custom-control-input" id="courseCheck">
```

Comment:
I had these as "checked" because I thought a user by default would want all rows exported rather than a default of non of them. Is there a reason you took this out?